### PR TITLE
feat: add builder projects panel controller

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

**Review by Suduck — APPROVED** [review]
LGTM! Small changes requested but approved

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/ScenesController/SceneData.cs [hunk 1]
```diff
@@ -77,4 +77,52 @@ internal class SceneData : ISceneData
...
+        bannedUsers = deployedScene.bannedUsers;
+        isEditable = deployedScene.source != DeployedScene.Source.SDK;
+        parcels = deployedScene.parcels;
+        projectId = deployedScene.projectId;
+
+        isMatureContent = false;
+        if (!string.IsNullOrEmpty(deployedScene.contentRating))
+        {
+            isMatureContent = deployedScene.contentRating.Equals("M", StringComparison.OrdinalIgnoreCase)
+                              || deployedScene.contentRating.Equals("AO", StringComparison.OrdinalIgnoreCase);
```
[the remark this answers is not in the document]
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/ScenesController/SceneData.cs]: done https://github.com/decentraland/unity-renderer/pull/369/commits/378b761e9c484e7fbfc4994ee52249b3067a6d57
There is no documentation about the content rating system, from Decentraland's docs we may assume ESRB
https://en.wikipedia.org/wiki/Video_game_content_rating_system#North_America so "AO" would mean "Adults only"

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Editor/BuilderProjectsPanelLocalTest.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        WebRequestController.Create();
+
+        controller = new BuilderProjectsPanelController();
+    }
+    void Start()
+    {
+        if (EventSystem.current == null)
+        {
+            var eventSystem = new GameObject("EventSystem", typeof(EventSystem), typeof(StandaloneInputModule));
```
[the remark this answers is not in the document]
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Editor/BuilderProjectsPanelLocalTest.cs:23]: Being code for testing (and something we are no repeating anywhere else in the code) I would prefer to keep that as explicit as possible

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/BuilderProjectsPanelController.cs [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+using DCL.Interface;
+using UnityEngine;
+using Variables.RealmsInfo;
+using Environment = DCL.Environment;
+using Object = UnityEngine.Object;
+
+public class BuilderProjectsPanelController : IHUD
+{
+    private const string TESTING_ETH_ADDRESS = "0x2fa1859029A483DEFbB664bB6026D682f55e2fcD";
+    private const string TESTING_TLD = "org";
```
[the remark this answers is not in the document]
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/BuilderProjectsPanelController.cs:15]: The idea for that is to be able to test the panel in editor with any address you like without the need to actually owning that address. That's my eth address if you want to send some crypto there to invite me a beer 🤫 🤣

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/BuilderProjectsPanelController.cs [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+#if UNITY_EDITOR
+        // NOTE: to be able to test in editor without getting a profile we hardcode an address here
+        if (string.IsNullOrEmpty(address))
+        {
+            address = TESTING_ETH_ADDRESS;
+            tld = TESTING_TLD;
+            DataStore.i.playerRealm.Set(new CurrentRealmModel()
+            {
+                domain = $"https://peer.decentraland.{TESTING_TLD}",
+                contentServerUrl = $"https://peer.decentraland.{TESTING_TLD}/content",
```
[the remark this answers is not in the document]
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/BuilderProjectsPanelController.cs:145]: Being those for testing in editor only, I don't see any reason on why we should create a constant for that... I could change it if you strongly disagree with that, but I always like to keep my code for testing as explicit as possible

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/BuilderProjectsPanelView.cs [hunk 5]
```diff
@@ -1,54 +1,138 @@
...
+    {
+        if (!isDestroyed)
+        {
+            Destroy(gameObject);
+        }
+    }
 
     private void Awake()
     {
+        name = "_BuilderProjectsPanel";
```
[the remark this answers is not in the document]
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/BuilderProjectsPanelView.cs:115]: I'm afraid I don't understand what you mean here 🤣
I'm setting there the name for the game object to keep the name structure that we use for game objects of huds that are instantiated on runtime

**Review by pravusjif — APPROVED** [review]

**Review by Suduck — APPROVED** [review]

**Pull request merged**